docs/user/metal/install_ipi.md

```diff
@@ -0,0 +1,225 @@
+    will be made configurable in the future.
+
+* **Out-of-band Management Network**
+  * Servers will typically have an additional NIC used by the onboard
+    management controllers (BMCs).  These BMCs must be accessible and routed to
+    the host.
+
+### Provisioning Host
+
+The installer must be run from a host that is attached to the same networks as
+the cluster, as described in the previous section.  We refer to this host as
+the *provisioning host*.  The easiest way to provide a provisioning host is to
+use one of the hosts that is intended to later become a worker node in the same
+cluster.  That way it is already connected to the proper networks.
+
+It is recommended that the provisioning host be a bare metal host, as it must be
+able to use libvirt to launch the OpenShift bootstrap VM locally.
+
+### Supported Hardware
+
+The architecture is intended to support a wide variety of hardware.  This was
+one of the reasons Ironic is used as an underlying technology.  However, so far
+development and testing has focused on PXE based provisioning using IPMI for
+out-of-band management of hosts.  Other provisioning approaches will be added,
+tested, and documented over time.
+
+## Installation Process
+
+Once an environment has been prepared according to the documented
+pre-requisites, the install process is the same as other IPI based platforms.
+
+`openshift-install create cluster`
+
+However, it is recommended to prepare an `install-config.yaml` file in advance,
+containing all of the details of the bare metal hosts to be provisioned.
+
+### Install Config
+
+The `install-config.yaml` file requires some additional details.  Most of the
+information is teaching the installer and the resulting cluster enough about
+the available hardware so that it is able to fully manage it.
+
+Here is an example `install-config.yaml` with the required `baremetal` platform
+details.
+
+**IMPORTANT NOTE:** The current install configuration for the `baremetal`
+platform should be considered experimental and still subject to change without
+backwards compatibility.  In particular, some items likely to change soon
+include:
+
+* The `image` section will get completely removed.
+
+* The `hardwareProfile` is currently exposed as a way to allow specifying
+  different hardware parameters for deployment.  By default, we will deploy
+  RHCOS to the first disk, but that may not be appropriate for all hardware.
+  The `hardwareProfile` is the field we have available to change that.  This
+  interface is subject to change.  In the meantime, hardware profiles can be
+  found here:
+  https://github.com/metal3-io/baremetal-operator/blob/master/pkg/hardware/profile.go#L48
+
+```yaml
+apiVersion: v1beta4
+baseDomain: test.metalkube.org
+metadata:
+  name: ostest
+compute:
+- name: worker
+  replicas: 1
+controlPlane:
+  name: master
+  replicas: 3
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: 192.168.111.5
```

Comment:
do you also need to specify the dnsVIP and the ingressVIP? Where does that happen?

Reply:
Good question.  Those should be documented.  There's code for getting their default values from DNS, so that's why this example worked.  The DNS VIP in particular really needs to be set, as we don't want to require a DNS record for that one.

Reply:
how about ironicuri and libvirturi, do we want to show all of the baremetal platform tunables here, or one some of them?

Reply:
`ironicuri` will go away shortly once it runs in the bootstrap VM.

`libvirturi` has a sane default, and I'm not sure even needs to be tunable, so I think leaving it out for now is OK

Reply:
`dnsVIP` isn't actually supported yet from config ...

Reply:
added `ingressVIP` to doc here: https://github.com/openshift-metal3/kni-installer/pull/163

Reply:
Strictly speaking neither apiVIP nor ingressVIP are required. In most circumstances they can be retrieved from DNS. dnsVIP cannot, however, so that will be mandatory. As Russell noted though, that change hasn't merged yet. I will go add it to the docs in that PR.